### DISCUSSION TITLE
FileConverter doublicated

### DISCUSSION THREAD

**Discussion by davethehacker** [microsoft/winget-pkgs#3269] [General]
# Brief description of your issue

The package "FileConverter" by Adrien Allard has two separate manifests in the repo and therefore two entries for `winget search FileConverter`. These manifests link to the same installer and therefore one entry should be removed.
https://github.com/microsoft/winget-pkgs/blob/master/manifests/AdrianAllard/FileConverter/1.2.3.yaml
https://github.com/microsoft/winget-pkgs/blob/master/manifests/FileConverter/FileConverter/1.2.3.yaml
# Steps to reproduce
`winget search FileConverter`

# Expected behavior
only one entry for the same installer

# Actual behavior
two different entries for the same installer

# Environment

```none
[winget --info]
Windows Package Manager version v0.1.42241 Vorschau
Windows: Windows.Desktop version v10.0.19041.450
Package: Microsoft.DesktopAppInstaller version  v1.10.42241.0

Any other software?
```

**Comment by davethehacker** [comment]
I would have just made a pull request, but I don't feel entitled to decide which one should be removed. I also think, there should be a process which checks new manifests against the already existing entries before merging to avoid duplicates like these.

**Comment by denelon** [comment]
We've seen a couple of duplicates, and we don't want to "break" users who may be depending on these packages. We're looking at checking for duplicate SHA256 values to identify duplicates at PR time. We're also looking at ways to have a manifest "redirect" to the preferred one in the event of a duplicate or a name change for the future. Other considerations include looking at installers that are no longer available so we can make sure users are informed if they try to install them. Feel free to start a GitHub [discussion](https://github.com/microsoft/winget-pkgs/discussions) if you'd like to join in the conversation.
